pom.xml

```diff
@@ -80,14 +80,6 @@
             <version>2.6.1</version>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/commons-io/commons-io -->
-       <dependency>
-            <groupId>commons-io</groupId>
-            <artifactId>commons-io</artifactId>
-            <version>[2.7,2.10)</version>
-       </dependency>
```

Comment:
Is this dependency no more required?

Reply:
It's a dependency, but it gets packaged with Atlassian Bamboo so I guess we needed to change the scope to "provided" after updating the AMPS version.